packages/modelviewer.dev/examples/lighthouse2.html

```diff
@@ -62,6 +62,69 @@
       display: none;
     }
 
+    .center {
+      display: flex;
+      justify-content: center;
+      align-items: center; 
+      height: 100%;
+      left: 0;
+      right: 0;
+      top: 0;
+      bottom: 0;
```

Comment:
According to this: https://css-tricks.com/almanac/properties/t/top-right-bottom-left/#setting-opposite-sides

if you have (top and left) set (right and bottom) are usually ignored.